Lib/test/test_argparse.py

```diff
@@ -471,8 +479,7 @@ class TestOptionalsAlternatePrefixCharsMultipleShortArgs(ParserTestCase):
                 '-xyz',
                 '+x',
                 '-y',
-                '+xyz',
-    ]
+                '+xyz']
```

Comment:
I think the original code is more idiomatic.

Reply:
I agree with @SylvainDe